support-frontend/assets/pages/[countryGroupId]/checkout.tsx

```diff
@@ -96,11 +96,12 @@ import { setThankYouOrder, unsetThankYouOrder } from './thank-you';
 /** App config - this is config that should persist throughout the app */
 validateWindowGuardian(window.guardian);
 
-const isTestUser = true as boolean;
 const csrf = window.guardian.csrf.token;
 
 const user = getUser();
 const isSignedIn = user.isSignedIn;
+const isTestUser = isTestUserFunc();
```

Comment:
Why not just call `isTestUser` direct?

Reply:
Yea, couldn't quite think of a better naming as the `const` is `isTestUser` which is also the method...